packages/web/integrations/axelar/hooks/use-transfer-fee-query.ts

```diff
@@ -0,0 +1,78 @@
+import { useState, useEffect } from "react";
+import { Environment, AxelarQueryAPI } from "@axelar-network/axelarjs-sdk";
+import { CoinPretty } from "@keplr-wallet/unit";
+import { AppCurrency } from "@keplr-wallet/types";
+
+/** Fetches a new Transfer fee quote when either chain, the amount, or the currency changes.
+ *  `amountMinDenom` is from user input, assumes `=== ""` for no input, and therefore no query.
+ */
+export function useTransferFeeQuery(
+  sourceChain: string,
+  destChain: string,
+  tokenMinDenom: string,
+  amountMinDenom: string,
+  currency: AppCurrency,
+  environment = Environment.MAINNET,
```

Comment:
should we also allow for the TESTNET environment? The values are not equal, and the TESTNET values appear to be completely erased.

Reply:
If we pass in Environment.TESTNET we should have the testnet values

https://github.com/milesz8/osmosis-frontend/blob/5443d2e5d3b525c5309cb4d926dd55e0e3d2c3fd/packages/web/integrations/axelar/transfer.tsx#L183